### ISSUE TITLE
restarting the times [exnos 7870], J5 and J7 2017v214

### ISSUE THREAD

**Issue by luizantxx** [phhusson/treble_experimentations#1199]
installation and startup is normal, but every time it gets restarted

**Comment by phhusson** [comment]
Sorry I don't understand what you're saying the issue is

**Comment by luizantxx** [comment]
Constant reboots, every now and then it restarts on its own.

**Comment by phhusson** [comment]
Right, that's sadly a known issue on many exynos devices running Pie
vendor, I have no fix for this.

**Comment by luizantxx** [comment]
right, v210 works well without rebooting and on its own, started from v211.  
Thanks for listening.

**Comment by phhusson** [comment]
Ok, that's probably another issue then.
Give logs.

**Comment by luizantxx** [comment]
[alogcat.2020-03-27-20-39-04-0300.txt](https://github.com/phhusson/treble_experimentations/files/4396345/alogcat.2020-03-27-20-39-04-0300.txt)
Here 😁

**Comment by luizantxx** [comment]
[alogcat.2020-03-27-20-54-27-0300.txt](https://github.com/phhusson/treble_experimentations/files/4396364/alogcat.2020-03-27-20-54-27-0300.txt)

**Comment by luizantxx** [comment]
[alogcat.2020-03-27-20-39-04-0300.txt](https://github.com/phhusson/treble_experimentations/files/4396365/alogcat.2020-03-27-20-39-04-0300.txt)
